pico_w/wifi/access_point/picow_access_point.c

```diff
@@ -96,6 +96,9 @@ int main() {
         return 1;
     }
 
+    // Initialise led state
+    led_state = status_led_get_state();
```

Comment:
I think there also needs to be a call to either [`status_led_init`](https://www.raspberrypi.com/documentation/pico-sdk/high_level.html#group_pico_status_led_1gaf83057f40dddf57bbaa9845c2c56201c) or [`status_led_init_with_context`](https://www.raspberrypi.com/documentation/pico-sdk/high_level.html#group_pico_status_led_1ga1d39f6237f4c2eb0b7d89938f379bd41) ? (Sorry I forgot to think of this earlier :facepalm: )